core/src/main/scripts/importer/validateData.py

```diff
@@ -2655,30 +2655,51 @@ def checkLine(self, data):
                                    'column_number': col_index + 1,
                                    'cause': value})
             elif col_name == 'OS_STATUS':
-                if value == 'DECEASED':
+                if value == '1:DECEASED':
                     osstatus_is_deceased = True
                 elif (value.lower() not in self.NULL_VALUES and
-                        value not in ('LIVING', 'DECEASED')):
+                        value not in ('LIVING', 'DECEASED', '0:LIVING', '1:DECEASED')):
```

Comment:
Should we pull these values out and put into a file for reading into the validator?  I thought we discussed something like this during the planning call.

Reply:
Do you mean for survival data migration tool? As Ramya suggested, I will put the vocabularies in a separate file for the migration tool. I can't really remember we have discussion about this... (please remind me if I missed something...)

Reply:
You remember correctly, we did discuss as it relates to survival data migration tool.  I'm suggesting we use a similar approach to inject the vocabulary here.  Maybe you are only validating OS_STATUS and DFS_STATUS and nothing more?

Reply:
Yes, we are just checking the `OS_STATUS` and `DFS_STATUS` here, for other values. For other status, we require all the values have the prefix `0:` and `1:` https://github.com/cBioPortal/cbioportal/blob/86f066aca39614fc7d7978e746d95d391b34162c/core/src/main/scripts/importer/validateData.py#L2688-L2702